v201705/campaign.go

```diff
@@ -122,25 +122,25 @@ type CustomParameters struct {
 type Campaign struct {
 	Id                             int64                           `xml:"id,omitempty"`
 	Name                           string                          `xml:"name"`
-	BudgetAmount                   int64                           `xml:"budget>amount>microAmount,omitempty"`
-	Status                         string                          `xml:"status"`                  // Status: "ENABLED", "PAUSED", "REMOVED"
+	BudgetAmount                   int64                           `xml:"-"`
+	Status                         string                          `xml:"status,omitempty"`        // Status: "ENABLED", "PAUSED", "REMOVED"
 	ServingStatus                  *string                         `xml:"servingStatus,omitempty"` // ServingStatus: "SERVING", "NONE", "ENDED", "PENDING", "SUSPENDED"
-	StartDate                      string                          `xml:"startDate"`
-	EndDate                        *string                         `xml:"endDate,omitempty"`
+	StartDate                      string                          `xml:"startDate,omitempty"`
+	EndDate                        string                          `xml:"endDate,omitempty"`
```

Comment:
Make sure you're running `go fmt` on all this code. It should remove the space after `string` here.